### PR TITLE
User feedback added on KT

### PR DESCRIPTION
**Pull request by bpurkaystha**
### Description

- Kafka tutorials website is supposed to be having sticky feedback button across all the pages
- Here's the Asana story for it: https://app.asana.com/0/1201817272699202/1202477897051080
- Here's the Figma design: https://www.figma.com/file/qMdrT2qOlnwRoudtHF3ouA/Developer.io?node-id=6698%3A35323

### CODE REVIEW

**Comment by ybyzek** [comment]
@bpurkaystha can someone from webdev review this first to ensure the requirement is met?

**Comment by bishwa420** [comment]
@ybyzek , I will ask someone from the web team. Thanks

**Comment by samuelleach** [comment]
Hello @bishwa420 

Following on from our chat a few weeks ago.

* [Bug] I'm finding that tapping on the feedback tab on mobile safari (latest iOS and Safari) doesn't do anything. Expect result: Feedback modal opens.
* Can you update the modal text to match the design: `How would you rate your experience` -> `Feedback`. I think this may need to be done via the Userback portal.

CleanShot 2022-07-25 at 11 40 02@2x

This is the preview I'm looking at https://deploy-preview-2234--confluent-developer.netlify.app/what-is-apache-kafka/ (let me know if there is a newer one).

**Comment by bishwa420** [comment]
Hii Sam,

I remember you telling me this. During development & testing, I was only
checking from mobile view, then there was no problem. Now it appears that
there could be a CORS issue. Does docs.confluent.io too have the same
issue? Did you manage some time to check it?

**Comment by samuelleach** [comment]
Hi @bishwa420 Just test docs userback feedback on mobile iOS safari and it works fine (docs userback is in production).

eg https://docs.confluent.io/home/connect/self-managed/userguide.html

**Comment by bishwa420** [comment]
@samuelleach , I believe you are referring to CD not KT. Anyhow, I will discuss it with the team today..

**Comment by bbejeck** [comment]
Staged at http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/user-feedback-staging

File: assets/sass/feedback.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  transform: translateX(-50%) !important;
+  background: #257bd7;
+  color: #ffffff;
+  border-radius: 0;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  padding: 12px 30px;
+  z-index: 99;
+  border: none;
+  font-weight: 700;
```
Comment (danshafie) [inline assets/sass/feedback.scss:14]: nit - i think cursor: pointer is nice ux

**Review by danshafie — APPROVED** [review]

**Review by bbejeck — APPROVED** [review]
Approval based on @danshafie

**Pull request merged**